tests/test_base_loader_integration.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import os
+import sys
+import uuid
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from loader import base_loader
+from tests.abstract_loader_test import AbstractLoaderTest
+
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+
+
+class TestBaseLoaderIntegration(AbstractLoaderTest):
+    """Integration tests for base_loader.py."""
```

Comment:
I just noticed that a lot of this setup / teardown stuff is copied between the two classes. If it makes sense to move, it would be more maintainable if they were consolidated, possibly in the parent class.